### DISCUSSION TITLE
Livewire render 'forgets' details/summary state

### DISCUSSION THREAD

**Discussion by FaycalBorsali** [livewire/livewire#4611] [3. Feature ideas]
When a livewire components has  `<details> <summary>` tags and at least one of them is open, if the component is re-rendered, the all tags are closed. However, I think it would be preferable to have them keep their state.

### Reproduce 
I created a [laravel playground snippet](https://laravelplayground.com/#/snippets/3066347c-0d20-4a9c-bc2d-b5f909041656) where the error can clearly be observed

1. Go to the link above
2. Open any of the `summary` tags
3. Click the button
4. **All tags are closed**

**Comment by joshhanley** [comment]
@FaycalBorsali ah interesting, it looks like the browser adds `open` to the details tag when it is open.

Yeah I think it would be good to support that, not sure how we would go about that though, as Livewire would need to know they have been opened as to not interfere with them on re-render.

I've changed this to a feature request.
